### PR TITLE
7.x.derivative rels

### PR DESCRIPTION
**Pull request by willtp87**

### CODE REVIEW

**Comment by willtp87** [comment]
Needs https://github.com/Islandora/islandora/pull/489
Needs https://github.com/Islandora/islandora_ocr/pull/40

File: includes/utilities.inc [hunk 1]
```diff
@@ -705,17 +705,53 @@ function islandora_paged_content_get_page_metadata_from_djatoka($object_id) {
...
 /**
+ * Returns paged object derivatives.
+ *
+ * @param string $context
+ *   Indicator of which solution pack calling this function
+ *
+ * @return array
+ *   Structured array of paged object derivatives
+ */
+function islandora_paged_content_paged_object_derivatives($context) {
```
Comment (nigelgbanks) [inline includes/utilities.inc:716]: Unused parameter

File: includes/utilities.inc [hunk 2]
```diff
@@ -705,17 +705,53 @@ function islandora_paged_content_get_page_metadata_from_djatoka($object_id) {
...
+  $derive = islandora_paged_content_get_enabled_ingest_derivatives('islandora_book');
```
Comment (willtp87) [inline includes/utilities.inc]: Bad copy paste on my part, good catch.

File: includes/utilities.inc (hunk 1 above)
Comment (willtp87) [inline includes/utilities.inc:716]: Should be good now.

**Pull request merged**